Tests/SemanticVersionTests/SemanticVersionTests.swift

```diff
@@ -124,7 +124,7 @@ final class SemanticVersionTests: XCTestCase {
         XCTAssert(SemanticVersion(1, 0, 0) < SemanticVersion(1, 1, 0))
         XCTAssert(SemanticVersion(1, 0, 0) < SemanticVersion(1, 0, 1))
         XCTAssert(SemanticVersion(1, 0, 0, "a") < SemanticVersion(1, 0, 0, "b"))
-        XCTAssert(SemanticVersion(1, 0, 0, "a", "a") < SemanticVersion(1, 0, 0, "a", "b"))
+        XCTAssertFalse(SemanticVersion(1, 0, 0, "a", "a") < SemanticVersion(1, 0, 0, "a", "b"))
```

Comment:
This test would probably be clearer now if it was

```swift
XCTAssertEqual(SemanticVersion(1, 0, 0, "a", "a"), SemanticVersion(1, 0, 0, "a", "b"))
```

(which might seem a little odd in a `Comparable` test but I think it makes sense)

Reply:
Oh wait, I think that's actually wrong. I was thinking the build id was being ignored but that's just for the `isStable` property. Ignore that last comment!